packages/payments/src/channel-manager.ts

```diff
@@ -402,7 +414,7 @@ export class ChannelManager implements ChannelManagementAPI {
 
     let results = await this.exchangeMessagesUntilOutboxIsEmpty(initialMessage);
 
-    for (const retryTimeoutMs of [2_500, 5_000, 10_000, 20_000, 40_000]) {
+    for (const retryTimeoutMs of [1_000, 2_000, 3_000, 5_000, 8_000, 13_000, 21_000]) {
```

Comment:
No reason other than they were originally arbitrary and I felt like adding slightly more timeouts without increasing the total length of the call seemed reasonable based on my gut feeling about the types of scenarios that would require timeout retries.

Reply:
Fibbonicci numbers are asymptotically exponential. `F_n+1 / F_n` quite quickly approaches ~1.6, actually, so these back-offs are essentially `range(6).map(i => 1_000 * 1.6 ** i)`.

I plan to make this configurable (eg. [here](https://github.com/statechannels/graph-payments/pull/30/commits/c842e9e8c5400755eb09fee625434e670b2a2128#diff-77cb416992eb54fd561f53ddd5bcaa1909db4e85b0a9a9d62682bf05faf9a603R186-R188)) so it can become `range(NUM).map(i => INITIAL_BACKOFF * RATIO ** i)`. This
1. allows us to test the behaviour of "retry attempts exceeded" in a sensible amount of time
2. allows the gateway to be able to decide how many retries, and how quickly